LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
There are ways in which we think the AGPL was the wrong decision for the platform. I don't want to be the final word on this if this is a significant policy change, I thought we were moving towards Apache in general because the AGPL is overly limiting.